### PR TITLE
doc(securing-your-webhooks): add standard JavaScript example

### PR DESCRIPTION
**Pull request by coolaj86**
Adds a JavaScript example that is secure, and follows the WebCrypto standards to work in all JavaScript environments.

### Why:

To add a cross-runtime JavaScript example

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Adds a cross-runtime JavaScript example

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@coolaj86 Thanks so much for submitting a PR! I'll get this triaged for review :zap:

File: content/webhooks-and-events/webhooks/securing-your-webhooks.md [hunk 1]
```diff
@@ -112,6 +112,57 @@ def verify_signature(payload_body, secret_token, signature_header):
         raise HTTPException(status_code=403, detail="Request signatures didn't match!")
 ```
 
+### JavaScript example
+
+For example, you can define the following `verifySignature` function and call it in any JavaScript environment - Node.js, Bun, Deno, etc - when you receive a webhook payload:
```
Comment (skedwards88) [inline content/webhooks-and-events/webhooks/securing-your-webhooks.md:117]: I think it's ok to remove this part to keep this streamlined
```suggestion
For example, you can define the following `verifySignature` function and call it in any JavaScript environment when you receive a webhook payload:
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for adding this! I'm going to incorporate one minor suggestion, then we'll get this merged for you.

**Review by skedwards88 — APPROVED** [review]
Thanks for adding this! I'm going to incorporate one minor suggestion, then we'll get this merged for you.

**Comment by peterbe** [comment]
@skedwards88 @cmwilson21 
The Azure preview environment deployment will start to work again once https://github.com/github/docs-internal/pull/40540 has landed in docs-internal and later been repo-sync'ed to this repo.

**Pull request merged**